### PR TITLE
fix: remove validation in operation, update message

### PR DESCRIPTION
**Pull request by alexcodelf**
**Problem:**

- validation status failed as request of resource does not pass status.
- modify dependency error message
- query stopping resource condition error

**Solution:**

- remove validation in operation upgrade
- update query conditon in  resource run dependant check

**Related Issue:**

#1461 #2155

### CODE REVIEW

File: pkg/resources/operation.go [hunk 1]
```diff
@@ -238,11 +238,6 @@ func upgrade(
 	}
 
 	if opts.Draft {
-		// Only stopped resource can save as draft.
-		if !status.ResourceStatusStopped.IsTrue(entity) {
```
Comment (gitlawr) [inline pkg/resources/operation.go:242]: Why remove it?

Comment (alexcodelf) [reply pkg/resources/operation.go:242]: The input resource comes from req.Model() which status is empty, can not using this to validation.

**Review by gitlawr — DISMISSED** [review]
LGTM

**Review by aiwantaozi — APPROVED** [review]
LGTM

**Pull request merged**